### PR TITLE
Messages for DCF algorithm

### PR DESCRIPTION
**Pull request by noether**
Btw,

is there any plan for having a "v2.0" in message_definitions? or similar. Now I was doubting where to put these new messages since they are meant for working only with v2.0 (air-to-air).

### CODE REVIEW

**Comment by noether** [comment]
I forgot to add descriptions... I will do it xD

File: message_definitions/v1.0/messages.xml [hunk 1]
```diff
@@ -2463,6 +2471,13 @@
       <field name="data" type="uint8[]"/>
     </message>
 
+    <message name="DCF_REG_TABLE" id="158" link="forwarded">
+      <description>Init the table of an aircraft for the Distributed Circular Formation algorithm</description>
+      <field name="ac_id" type="uint8">ID of the table's aircraft to be updated</field>
+      <field name="nei_id" type="uint8">ID of the neighborh for the table</field>
+      <field name="desired_sigma" type="int16"/>Desired angle to have w.r.t. the neighborh</field>
```
Comment (gautierhattenberger) [inline message_definitions/v1.0/messages.xml]: you are closing the field with `/>` too early

File: message_definitions/v1.0/messages.xml [hunk 2]
```diff
@@ -1982,7 +1986,11 @@
       <field name="bus_number"                  type="uint8"/>
     </message>
 
-  <!-- 254 is free -->
+    <message name="DCF_THETA" id="254">
+      <description>Air-to-air message for the Distributed Circular Formation algorithm. It transmits the ac's theta to its neighborg
```
Comment (gautierhattenberger) [inline message_definitions/v1.0/messages.xml]: spelling error: neighborg

File: message_definitions/v1.0/messages.xml [hunk 3]
```diff
@@ -338,7 +338,11 @@
       <field name="remote_noise" type="uint8"/>
     </message>
 
-    <!-- 40 is free -->
+    <message name="DCF" id="40">
+      <description>Telemetry message for monitoring the status of the Distributed Circular Formation</description>
+      <field name="table" type="int16[]"> 1. ID of the neighbor, 2. Theta of the neighbor (degrees x 100), 3. Desired inter-vehicle angle (degrees x 100), 4. Last time in ms we received a msg from the neighbor</field>
```
Comment (gautierhattenberger) [inline message_definitions/v1.0/messages.xml]: do you know the size of the table in advance (e.g. the 4 elements described) or does it depend on the number of neighbors (n x 4 elements) ?

File: message_definitions/v1.0/messages.xml [hunk 4]
```diff
@@ -2463,6 +2471,13 @@
       <field name="data" type="uint8[]"/>
     </message>
 
+    <message name="DCF_REG_TABLE" id="158" link="forwarded">
+      <description>Init the table of an aircraft for the Distributed Circular Formation algorithm</description>
+      <field name="ac_id" type="uint8">ID of the table's aircraft to be updated</field>
+      <field name="nei_id" type="uint8">ID of the neighborh for the table</field>
```
Comment (gautierhattenberger) [inline message_definitions/v1.0/messages.xml]: are you going to use the `nei_id = 0` to clear the table in the end ? If yes it should be in the description of the field

**Comment by noether** [comment]
okii, done.

**Comment by gautierhattenberger** [comment]
additionally, the version of the messages are not related to the version of library itself.
I do have plans to refine and clean the messages in several classes for air-ground and air-air part, but not much time.

**Pull request merged**